docker-1.13.0-changelog.md

```diff
@@ -0,0 +1,113 @@
+# Docker 1.13 发布，编排领域迈向成熟期
+
+## 镜像构建器
+
+* 在镜像构建时，允许用户指定镜像，将该镜像作为构建的缓存源。这些镜像不需要拥有本地的父子链关系，同事可以通过从其他惊喜那个仓库中下啦
+* 在镜像构建成功之后，可以添加一个参数来压缩镜像层中From的基础镜像（试验版功能）
+* 修复了Dockerfile解析器的Bug，该bug在程序退出时存在空行的时候出现
+* 在`docker build`命令中添加了构建步数
+* 镜像构建时，添加压缩构建上下文的支持
+* 在`docker build`命令中添加`--network`参数，使镜像构建时允许指定具体网络
+* 修复构建镜像时的`--label`和 `docker run`时`--label`参数的不一致性
+* 修复了使用overlay存储驱动时，镜像层可能存在的不一致性
+* 之前没有使用的`build-args`目前已经支持，届时会显示一条警告日志，而不是使整个构建流程报错退出
+* 修复了Windows系统上构建缓存的问题
+* 允许在Windows系统构建时支持User参数
+* 在Windows系统上修复了环境变量大小写的问题
+
+## 发行版
+
+* 为 PPC64LE 上的 Ubuntu 16.04 Xenial 支持构建docker包
+* 为 s390x 的 Ubuntu 16.04 Xenial 支持构建docker包
+* 为 PPC64LE 的 Ubuntu 16.10 Yakkety Yak 支持构建docker包
+* 为 VMWare Photon OS 支持docker的RPM构建
+* 为tgz添加shell补全功能
+* 为Docker的安装脚本添加中国的镜像源
+* 为 Ubuntu 16.10 Yakkety Yak 系统添加DEB包的构建
+* 为 Fedora 25 添加RPM包de构建
+* 为 aarch64 添加 `make deb` 的支持
+
+## 镜像仓库 Distribution
+
+* 更新 notary 依赖到0.4.2版本
+  - 支持Windows系统上的编译
+  - 优化客户验证错误
+  - 支持寻找在 `~/.docker/trust/private` 下任意位置的 key, 不仅仅是在目录 `~/.docker/trust/private/root_keys` 或者 `~/.docker/trust/private/tuf_keys`
+  - 之前notary的实现，对于任何的更新错误，客户端都会在缓存上失效。现在，我们只有在网络错误，或者服务器不响应服务，或者修饰TUF数据时返回错误。无效的TUF会导致更新失败，例如用户发起了一个无效的root滚动更新。
+  - 改进root验证以及yubikey debug 日志
+  - 当root验证或者快过期时发出警告
+  - 当角色元数据临近过期时发出警告
+  - 修复修复密码检索尝试计数和终端检测存在的问题
+* 当不同的用户上传同样的镜像层到受验证的镜像仓库时, 避免重复上传不必要的块
+* 允许镜像仓库认证信息的外部存储
+
+## 日志
+
+* 在所有的日志驱动中标准化默认的日志标签值；
+* 当使用长日志行数时，提高性能与内存利用率
+* 支持Windows系统上的syslog驱动
+* 添加Logentries驱动
+* 更新AWS日志驱动以支持日志标签
+* 为fluentd日志驱动添加unix socket的支持
+* 在Windows系统上支持fluentd
+* 当docker的标签被用作journald的名称时，审查这些labels标签
+* 修复了 `docker logs --tail` 显示的日志行数比实际少的错误
+* 为Splunk的日志驱动提高性能和可靠性
+* 为Splunk的日志驱动格式化配置项，同时跳过验证连接
+
+## 网络
+
+* 为每个网络添加`--attachable`参数，使得支持容器在运行时加入swarm mode的overlay网络中
+* 在`docker service create`中使用`--publish`选项时，添加了任务单主机端口暴露的功能
+* 为Windows server 2016添加overlay网络驱动的支持（需要即将到来的ws2016更新）
```

Comment:
Thanks for the feedback and provided reference. Updated as well.
While what makes me excited is that the reason why you are so familiar with Chinese. A Chinese? or interest in Chinese? 😄